### ISSUE TITLE
Post mental health signals from the symptom survey

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/covidcast-indicators#323]
We can calculate the following signals from the survey data:

* `anxious_5d`: Percent of people who report being nervous/anxious most or all of the time in the past 5 days
* `depressed_5d`: Percent of people who report being depressed most or all of the time in the past 5 days
* `felt_isolated_5d`: Percent of people who report feeling isolated from others most or all of the time in the past 5 days
*  `worried_become_ill`: Percent answering very or somewhat worried to item C9, "How do you feel about the possibility that you or someone in your immediate family might become seriously ill from COVID-19 (coronavirus disease)?"

However, we have not resolved with the IRB whether releasing these aggregates is within the bounds of our protocol. Once we do, if we can release them (or release only data collected at Wave 4 onward, for instance), we should write the docs and publish the signals.

Followup from #18.

**Comment by capnrefsmmat** [comment]
We've resolved the IRB issues and can release data collected from Wave 4 onward. I've implemented this in the code.

I'd also like to add the following signal:

* `worried_finances`: Percent answering very or somewhat worried to item C15, "How worried are you about your household's finances for the next month?" Facebook's Health team would like to do work and PR based on financial signals.

@RoniRos Any changes to these signal names before we publicly release them?
